content/id/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Batas Rentang (Limit Range)
+content_template: templates/concept
+weight: 10
+---
+
+<!-- overview -->
+
+Secara bawaan, Container berjalan dengan [sumber daya komputasi](id/docs/user-guide/compute-resources) tanpa batas pada klaster Kubernetes.
+Dengan kuota sumber daya, administrator klaster dapat membatasi konsumsi dan pembuatan sumber daya berbasis {{<glossary_tooltip text="namespace" term_id="namespace">}}.
+Di dalam _namespace_, Pod atau Container dapat mengkonsumsi CPU dan memori sesuai dengan yang ditentukan oleh kuota sumber daya _namespace_. Ada kekhawatiran bahwa satu Pod atau Container dapat memonopoli semua sumber daya yang tersedia. LimitRange adalah kebijakan untuk membatasi alokasi sumber daya (bagi Pod atau Container) pada _namespace_.
+
+
+<!-- body -->
+
+LimitRange memberikan batasan yang dapat:
+
+- Menerapkan penggunaan sumber daya komputasi minimum dan maksimum untuk setiap Pod atau Container dalam _namespace_.
+- Menerapkan permintaan tempat penyimpanan minimum dan maksimum untuk setiap PersistentVolumeClaim dalam _namespace_.
+- Menerapkan rasio antara permintaan dan batas untuk sumber daya dalam _namespace_.
+- Menetapkan permintaan/batas bawaan untuk menghitung sumber daya dalam _namespace_ dan memasukkannya secara otomatis ke Container pada _runtime_.
+
+## Mengaktifkan LimitRange
+
+Dukungan LimitRange diaktifkan secara bawaan untuk banyak distribusi Kubernetes. Hal ini
+diaktifkan ketika tanda `--enable-admission-plugins=` pada apiserver memiliki admission controller `LimitRanger` sebagai
+salah satu argumennya.
+
+LimitRange diberlakukan pada _namespace_ tertentu ketika ada sebuah objek LimitRange pada _namespace_ tersebut.
+
+Nama dari objek LimitRange harus merupakan sebuah [nama subdomain DNS](id/docs/concepts/overview/working-with-objects/names#dns-subdomain-names).
```

Comment:
missing `/` and incorrect anchor link
```suggestion
Nama dari objek LimitRange harus merupakan sebuah [nama subdomain DNS](/id/docs/concepts/overview/working-with-objects/names/#nama).
```

Reply:
we might haven't translated the new subsections in the DNS page

Reply:
Ok!